cl/recap/mergers.py

```diff
@@ -1753,6 +1753,16 @@ async def merge_attachment_page_data(
                 params["acms_document_guid"] = attachment["acms_document_guid"]
             try:
                 rd = await RECAPDocument.objects.aget(**params)
+            except RECAPDocument.MultipleObjectsReturned:
```

Comment:
I think the issue right now is that there are already two `PACER_DOCUMENT` records, so when this tries to remove the attachment_number and save it, the database raises a unique constraint violation on (docket_entry_id, document_number, attachment_number) because another `PACER_DOCUMENT` with `attachment_number=None` already exists. So, maybe the other PACER_DOCUMENT needs to be converted to an attachment first?